content/fewer-braces.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: sip
+permalink: /sips/:title.html
+stage: ready for review
+status: implemented
+title: SIP-NN Fewer Braces
+---
+
+**By: Martin Odersky**
+
+## History
+
+| Date          | Version            |
+|---------------|--------------------|
+| July 1st 2022 | Initial Draft      |
+
+## Summary
+
+The current state of Scala 3 makes braces optional around blocks and template definitions (i.e. bodies of classes, objects, traits, enums, or givens). This SIP proposes to allow optional braces also for function arguments.
+The advantages of doing so is that the language feels more systematic, and programs become typographically cleaner.
+The changes have been implemented and and made available under the language import `language.experimental.fewerBraces`. The proposal here is to make them available without a language import instead.
+
+
+## Motivation
+
+After extensive experience with the current indentation rules I conclude that they are overall a big success.
+However. they still feel incomplete and a bit unsystematic since we can replace `{...}` in the majority of situations, but there are also important classes of situations where braces remain mandatory. In particular, braces are currently needed around blocks as function arguments.
+
+It seems very natural to generalize the current class syntax indentation syntax to function arguments. In both cases, an indentation block is started by a colon at the end of a line.
+
+
+## Proposed solution
+
+The proposed solution is described in detail in https://dotty.epfl.ch/docs/reference/other-new-features/indentation.html#variant-indentation-marker--for-arguments.
```

Comment:
Do you mind inlining the relevant content here? That will make the review process simpler.